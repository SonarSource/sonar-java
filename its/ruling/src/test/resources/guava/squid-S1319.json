{
'com.google.guava:guava:src/com/google/common/collect/Lists.java':[
87,
109,
141,
158,
183,
206,
231,
256,
273,
285,
],
'com.google.guava:guava:src/com/google/common/collect/Maps.java':[
176,
193,
228,
242,
257,
290,
306,
320,
336,
349,
359,
],
'com.google.guava:guava:src/com/google/common/collect/Queues.java':[
61,
71,
85,
93,
242,
],
'com.google.guava:guava:src/com/google/common/collect/Sets.java':[
143,
163,
181,
200,
217,
236,
288,
306,
322,
343,
362,
380,
408,
420,
445,
468,
],
'com.google.guava:guava:src/com/google/common/collect/TreeBasedTable.java':[
89,
],
}
