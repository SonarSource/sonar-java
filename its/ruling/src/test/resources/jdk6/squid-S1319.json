{
'jdk6:java/awt/image/BufferedImage.java':[
608,
],
'jdk6:java/awt/image/MemoryImageSource.java':[
137,
177,
230,
],
'jdk6:java/util/Collections.java':[
3409,
],
'jdk6:java/util/EnumMap.java':[
123,
],
'jdk6:java/util/EnumSet.java':[
90,
109,
128,
128,
144,
167,
167,
186,
206,
228,
252,
278,
303,
325,
],
}
