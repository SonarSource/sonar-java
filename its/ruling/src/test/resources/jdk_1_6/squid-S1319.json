{
'jdk_1_6:java/awt/image/BufferedImage.java':[
608,
],
'jdk_1_6:java/awt/image/CropImageFilter.java':[
64,
],
'jdk_1_6:java/awt/image/ImageFilter.java':[
89,
],
'jdk_1_6:java/awt/image/MemoryImageSource.java':[
137,
177,
230,
],
'jdk_1_6:java/awt/image/PixelGrabber.java':[
393,
],
'jdk_1_6:java/awt/image/ReplicateScaleFilter.java':[
106,
],
'jdk_1_6:java/util/ArrayDeque.java':[
787,
],
'jdk_1_6:java/util/Collections.java':[
3409,
],
'jdk_1_6:java/util/EnumMap.java':[
123,
641,
],
'jdk_1_6:java/util/EnumSet.java':[
90,
109,
128,
128,
144,
167,
167,
186,
206,
228,
252,
278,
303,
325,
344,
],
'jdk_1_6:java/util/JumboEnumSet.java':[
347,
],
'jdk_1_6:java/util/concurrent/ConcurrentSkipListMap.java':[
1452,
],
}
