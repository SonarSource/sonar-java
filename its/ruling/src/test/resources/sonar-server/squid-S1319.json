{
'org.sonarsource.sonarqube:sonar-server:src/main/java/org/sonar/server/es/EsUtils.java':[
69,
],
'org.sonarsource.sonarqube:sonar-server:src/main/java/org/sonar/server/es/Facets.java':[
50,
156,
],
'org.sonarsource.sonarqube:sonar-server:src/main/java/org/sonar/server/issue/ws/SearchAdditionalField.java':[
71,
],
}
