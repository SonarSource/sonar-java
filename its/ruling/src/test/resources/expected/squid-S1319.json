{
'project:com/sun/corba/se/impl/naming/cosnaming/TransientBindingIterator.java':[
70,
],
'project:com/sun/corba/se/impl/naming/pcosnaming/PersistentBindingIterator.java':[
71,
],
'project:com/sun/corba/se/impl/orbutil/RepositoryId_1_3.java':[
183,
192,
200,
212,
223,
235,
246,
],
'project:com/sun/corba/se/impl/orbutil/RepositoryId_1_3_1.java':[
201,
210,
218,
230,
241,
253,
264,
],
'project:com/sun/corba/se/impl/presentation/rmi/JNDIStateFactoryImpl.java':[
103,
],
'project:com/sun/corba/se/impl/util/RepositoryId.java':[
195,
204,
212,
224,
235,
247,
258,
],
'project:com/sun/imageio/plugins/png/PNGMetadata.java':[
177,
178,
179,
180,
181,
182,
214,
215,
238,
239,
240,
243,
244,
],
'project:com/sun/jmx/remote/util/EnvHelp.java':[
775,
],
'project:com/sun/jmx/snmp/IPAcl/Host.java':[
113,
146,
],
'project:com/sun/jmx/snmp/IPAcl/JDMAclBlock.java':[
54,
60,
],
'project:com/sun/jmx/snmp/IPAcl/JDMInformBlock.java':[
59,
],
'project:com/sun/jmx/snmp/IPAcl/JDMTrapBlock.java':[
60,
],
'project:com/sun/jmx/snmp/IPAcl/SimpleNode.java':[
93,
106,
],
'project:com/sun/naming/internal/ResourceManager.java':[
186,
186,
247,
308,
391,
],
'project:com/sun/org/apache/xalan/internal/xslt/EnvironmentCheck.java':[
217,
413,
],
'project:com/sun/org/apache/xalan/internal/xsltc/DOM.java':[
106,
],
'project:com/sun/org/apache/xalan/internal/xsltc/compiler/Mode.java':[
247,
],
'project:com/sun/org/apache/xalan/internal/xsltc/dom/AdaptiveResultTreeImpl.java':[
559,
],
'project:com/sun/org/apache/xalan/internal/xsltc/dom/DOMAdapter.java':[
465,
],
'project:com/sun/org/apache/xalan/internal/xsltc/dom/MultiDOM.java':[
668,
],
'project:com/sun/org/apache/xalan/internal/xsltc/dom/SAXImpl.java':[
1845,
],
'project:com/sun/org/apache/xalan/internal/xsltc/dom/SimpleResultTreeImpl.java':[
611,
],
'project:com/sun/org/apache/xalan/internal/xsltc/runtime/AbstractTranslet.java':[
264,
704,
],
'project:com/sun/org/apache/xerces/internal/impl/dtd/XMLDTDDescription.java':[
135,
],
'project:com/sun/org/apache/xerces/internal/impl/dv/DTDDVFactory.java':[
85,
],
'project:com/sun/org/apache/xerces/internal/impl/dv/dtd/DTDDVFactoryImpl.java':[
57,
],
'project:com/sun/org/apache/xerces/internal/impl/dv/dtd/XML11DTDDVFactoryImpl.java':[
61,
],
'project:com/sun/org/apache/xerces/internal/impl/xs/models/XSAllCM.java':[
220,
],
'project:com/sun/org/apache/xerces/internal/impl/xs/models/XSCMValidator.java':[
109,
],
'project:com/sun/org/apache/xerces/internal/impl/xs/models/XSDFACM.java':[
1219,
],
'project:com/sun/org/apache/xerces/internal/impl/xs/models/XSEmptyCM.java':[
137,
],
'project:com/sun/org/apache/xerces/internal/util/DOMUtil.java':[
187,
244,
296,
322,
340,
359,
],
'project:com/sun/org/apache/xml/internal/serialize/ElementState.java':[
105,
],
'project:com/sun/org/apache/xml/internal/serializer/EmptySerializer.java':[
566,
],
'project:com/sun/org/apache/xml/internal/utils/Hashtree2Node.java':[
66,
],
'project:com/sun/security/auth/PolicyParser.java':[
653,
],
'project:java/awt/image/BufferedImage.java':[
625,
],
'project:java/awt/image/CropImageFilter.java':[
81,
],
'project:java/awt/image/ImageFilter.java':[
106,
],
'project:java/awt/image/MemoryImageSource.java':[
154,
194,
247,
],
'project:java/awt/image/PixelGrabber.java':[
410,
],
'project:java/awt/image/ReplicateScaleFilter.java':[
123,
],
'project:java/util/ArrayDeque.java':[
814,
],
'project:java/util/Collections.java':[
3677,
],
'project:java/util/EnumMap.java':[
140,
716,
],
'project:java/util/EnumSet.java':[
107,
126,
145,
145,
161,
184,
184,
203,
223,
245,
269,
295,
321,
342,
361,
],
'project:java/util/JumboEnumSet.java':[
370,
],
'project:java/util/concurrent/ConcurrentSkipListMap.java':[
1438,
],
'project:javax/management/MBeanServerFactory.java':[
358,
],
'project:javax/management/ObjectName.java':[
1330,
1423,
1623,
],
'project:javax/naming/CannotProceedException.java':[
160,
172,
],
'project:javax/naming/Context.java':[
780,
],
'project:javax/naming/InitialContext.java':[
210,
543,
],
'project:javax/naming/ReferralException.java':[
150,
],
'project:javax/naming/directory/InitialDirContext.java':[
98,
],
'project:javax/naming/ldap/ControlFactory.java':[
135,
],
'project:javax/naming/ldap/InitialLdapContext.java':[
132,
],
'project:javax/naming/ldap/LdapReferralException.java':[
117,
157,
],
'project:javax/naming/spi/ContinuationContext.java':[
196,
],
'project:javax/naming/spi/DirObjectFactory.java':[
129,
],
'project:javax/naming/spi/DirStateFactory.java':[
143,
],
'project:javax/naming/spi/DirectoryManager.java':[
155,
304,
],
'project:javax/naming/spi/InitialContextFactory.java':[
70,
],
'project:javax/naming/spi/InitialContextFactoryBuilder.java':[
69,
],
'project:javax/naming/spi/NamingManager.java':[
289,
544,
650,
853,
],
'project:javax/naming/spi/ObjectFactory.java':[
172,
],
'project:javax/naming/spi/ObjectFactoryBuilder.java':[
73,
],
'project:javax/naming/spi/StateFactory.java':[
136,
],
'project:javax/swing/JSlider.java':[
895,
923,
],
'project:javax/swing/JTree.java':[
610,
],
'project:javax/swing/text/html/parser/DTD.java':[
62,
64,
],
'project:javax/swing/undo/StateEditable.java':[
47,
53,
],
}
